ios/PurchasesHybridCommon/PurchasesHybridCommonUI/PaywallProxy.swift

```diff
@@ -185,6 +184,13 @@ import UIKit
 
     }
 
+    private func createDismissHandler() -> (PaywallViewController) -> Void {
+        return { [weak self] controller in
+            guard let delegate = self?.delegate else { return }
```

Comment:
Hmm thinking about it, we could also hold just the delegate (weakly) instead of self... But this should be fine.

Reply:
actually the delegate is already weak

    public weak var delegate: PaywallViewControllerDelegateWrapper?

Reply:
but yes, let's leave it like this, I tested it just in case and it works fine.